lib/node_modules/@stdlib/array/base/mskunary5d/test/test.js

```diff
@@ -0,0 +1,404 @@
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var zeros5d = require( '@stdlib/array/base/zeros5d' );
+var mskunary5d = require( '@stdlib/array/base/mskunary5d/lib' );
+
+
+// TESTS //
+
+tape('main export is a function', function test(t) {
+	t.ok(true, __filename);
+	t.strictEqual(typeof mskunary5d, 'function', 'main export is a function');
+	t.end();
+});
+
+tape('the function applies a provided callback to a nested input array and assigns results to a nested output array', function test(t) {
+	var expected;
+	var shape;
+	var x;
+	var y;
+	var m;
+
+	shape = [2, 2, 2, 2, 2];
+	x = [
+		[
+			[
+				[
+					[-4, -4], [-1, -6]
+				],
+				[
+					[-4, -4], [-1, -6]
+				]
+			],
+			[
+				[
+					[-4, -4], [-1, -6]
+				],
+				[
+					[-4, -4], [-1, -6]
+				]
+			]
+		],
+		[
+			[
+				[
+					[-4, -4], [-1, -6]
+				],
+				[
+					[-4, -4], [-1, -6]
+				]
+			],
+			[
+				[
+					[-4, -4], [-1, -6]
+				],
+				[
+					[-4, -4], [-1, -6]
+				]
+			]
+		]
+	];
+	m = [
+		[
+			[
+				[
+					[1, 0], [0, 0]
+				],
+				[
+					[0, 1], [0, 0]
+				]
+			],
+			[
+				[
+					[0, 0], [0, 0]
+				],
+				[
+					[0, 1], [1, 0]
+				]
+			]
+		],
+		[
+			[
+				[
+					[0, 0], [0, 0]
+				],
+				[
+					[0, 1], [0, 0]
+				]
+			],
+			[
+				[
+					[0, 0], [1, 0]
+				],
+				[
+					[0, 1], [1, 0]
+				]
+			]
+		]
+	];
+
+	expected = [
+		[
+			[
+				[
+					[0.0, 4.0], [1.0, 6.0]
+				],
+				[
+					[4.0, 0.0], [1.0, 6.0]
+				]
+			],
+			[
+				[
+					[4.0, 4.0], [1.0, 6.0]
+				],
+				[
+					[4.0, 0.0], [0.0, 6.0]
+				]
+			]
+		],
+		[
+			[
+				[
+					[4.0, 4.0], [1.0, 6.0]
+				],
+				[
+					[4.0, 0.0], [1.0, 6.0]
+				]
+			],
+			[
+				[
+					[4.0, 4.0], [0.0, 6.0]
+				],
+				[
+					[4.0, 0.0], [0.0, 6.0]
+				]
+			]
+		]
+	];
+
+	y = zeros5d(shape);
+	mskunary5d([x, m, y], shape, abs);
+
+	t.deepEqual(y, expected, 'returns expected value');
+
+	shape = [ 1, 1, 1, 2, 2 ];
+	x = [
+		[
+			[
+				[
+					[-1.0, -2.0],
+					[-3.0, -4.0]
+				]
+			]
+		]
+	];
+	m = [
+		[
+			[
+				[
+					[0, 1],
+					[0, 0]
+				]
+			]
+		]
+	];
+	expected = [
+		[
+			[
+				[
+					[1.0, 0.0],
+					[3.0, 4.0]
+				]
+			]
+		]
+	];
+	y = zeros5d(shape);
+	mskunary5d([x, m, y], shape, abs);
+	t.deepEqual(y, expected, 'returns expected value');
+	t.end();
+});
+tape( 'the function does not invoke a provided callback if provided a shape having a first element equal to zero', function test( t ) {
```

Comment:
```suggestion
});

tape( 'the function does not invoke a provided callback if provided a shape having a first element equal to zero', function test( t ) {
```
Should follow space indentation everywhere in this file.